compiler-cli/src/config.rs

```diff
@@ -12,6 +13,35 @@ pub fn root_config() -> Result<PackageConfig, Error> {
     read(paths.root_config())
 }
 
+/// Get the config for a dependency module. Return the config for the current
+/// project if a dependency doesn't have a config file.
+pub fn module_config(mod_path: &str, project_paths: &ProjectPaths) -> Result<PackageConfig, Error> {
```

Comment:
It wasn't clear to me what this function did until I looked at it for a while, so let's be more verbose with the naming.
```suggestion
pub fn find_package_config_for_module(module_name: &str, project_paths: &ProjectPaths) -> Result<PackageConfig, Error> {
```